code/game/gamemodes/gang/dominator.dm

```diff
@@ -48,6 +49,20 @@
 	if(gang && gang.is_dominating)
 		var/time_remaining = gang.domination_time_remaining()
 		if(time_remaining > 0)
+			var/open = 0
+			for(var/turf/T in circleviewturfs(center=src,radius=3))
+				if(!istype(T, /turf/closed))
```

Comment:
I tried that proc for a few other things and I quickly learned how common dense tiles are.

With almost everything destructable now I'm not that worried about most non-wall obstructions.